templates/minimal/README.md

```diff
@@ -1,13 +1,93 @@
-# Minimal Template
+<div align="center">
 
-This is a minimal template for creating a blockchain using the Polkadot SDK.
+# Polkadot SDK's Minimal Template
 
-# Docs
+<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_White.png#gh-dark-mode-only"/>
+<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_Black.png#gh-light-mode-only"/>
 
-You can generate and view the [Rust
-Docs](https://doc.rust-lang.org/cargo/commands/cargo-doc.html) for this template
-with this command:
+> This is a minimal template for creating a [Substrate](https://substrate.io/) blockchain.
```

Comment:
Can we already avoid linking to substrate.io overall? it is all future refactor if we leave it like this. 

I suppose for all of this we can either find a replacement, or bring it to `polkadot-sdk-docs` crate.

Reply:
OK, I'll try to find replacement for those substrate.io links.

Reply:
I have removed all `substrate.io` links. I also revamped the sub-readmes for Node, Pallets and Runtime, please take another look.